### DISCUSSION TITLE
Borderless windows

### DISCUSSION THREAD

**Discussion by 4jag** [standardnotes/forum#2900] [Suggestions]
**Describe the bug**
The Windows Desktop program seems to have Borderless/frameless Windows.
Can this be changed; is it configurable?

**To Reproduce**
1. Run Standardnotes on Windows
2. Become visually confused because Standardnotes has borderless windows.

**Expected behavior**
All application windows should allow the standard Windows UI to set windows frames styles, backgrounds, etc.
Maybe your developers have a single windows open, Standardnotes, because you're working very hard to write a cool piece of software (Thank you!), or you have multi or huge monitors and place windows side-by-side. But the rest of us, if anything like me, have 50 other windows open -- all stacked -- a few file managers, Acrobat, terminal windows, text editor, Thunderbird, and 40 Browser windows with 10 tabs each. If a windows-user wishes to have borderless windows then the can configure this via a Windows style tool. (Microsoft tried borderless, and everyone I know hates it. We all use Aero or similar to re-enable borders)
Forcing borderless windows is a NON-Starter for me. I hope it's configurable.

**Screenshots**
![image](https://user-images.githubusercontent.com/84933394/202756344-43322924-4a2e-4874-982d-fede2afe0099.png)

**Desktop (please complete the following information):**
 - OS: Windows10
 - Browser NA (Firefox)]
 - Version NA

**Comment by myreli** [comment]
Hi @4jag! You can disable the theme menu bar by toggling `Menu > View > Use Themed Menu Bar`

<img width="516" alt="image" src="https://user-images.githubusercontent.com/17554234/221389871-fd74173c-9a56-432d-9a36-4b012417a9ff.png">

After disabling you will have a standard OS title bar: 

<img width="1029" alt="image" src="https://user-images.githubusercontent.com/17554234/221389906-712faa15-fac1-488f-85ec-1f872e247598.png">
